site/developer/model-documentation/generate-model-documentation.qmd

```diff
@@ -1,128 +1,77 @@
 
-2. Export the datasets and model.
-
-Next, go to **With the {{< var validmind.developer >}}, Step 2**.  {{< fa arrow-right >}}
+2. Export the datasets and model or predictions.
 
 ### With the {{< var validmind.developer >}}
 
-1. Create a notebook to select and build the relevant tests.
-
-2. {{< fa arrow-right >}} **From your modeling environment**, load the trained datasets and models.
+1. **From your modeling environment**, load the trained datasets and models or predictions.
 
-3. {{< fa arrow-right >}} Use the instructions from **In the {{< var vm.platform >}}, Step 3**, [initialize the {{< var validmind.developer >}}](install-and-initialize-developer-framework.qmd).
+2. [Install and initialize ValidMind.](install-and-initialize-validmind.qmd)
 
-4. Select the relevant tests.
+3. Select the relevant tests.
 
-5. Review if all tests are covered by ValidMind or your external test provider:
+4. Review if all tests are covered by ValidMind or your external test provider:
     
-   - If all tests are NOT covered — Create and register additional tests.
-   - If all tests are covered — 
+   - **If all tests are NOT covered** — Create and register additional tests.
+   - **If all tests are covered** — 
       a. Run the selected tests.
       b. Review your test results. 
 
-Next, go to **In the {{< var validmind.platform >}}, Step 5**.  {{< fa arrow-right >}} 
-
 ### In the {{< var validmind.platform >}}
 
-1. [Register a new model](/guide/model-inventory/register-models-in-inventory.qmd).
-
-2. Review the template structure.
-
-3. [Locate the {{< var vm.developer >}} integration instructions](install-and-initialize-developer-framework.qmd#locate-the-framework-integration-instructions). 
-   
-4.  Go to **With the {{< var validmind.developer >}}, Step 3**. {{< fa arrow-right >}}
-
-5. {{< fa arrow-right >}} After **With the {{< var validmind.developer >}}, Step 6**, [add content blocks](/guide/model-documentation/work-with-content-blocks.qmd) to your model documentation: 
+1. **After installing and initalizing the {{< var validmind.developer >}}**,[^4] add content blocks[^3] to your model documentation: 
    
    Select the block type:
    
-      - For test-driven blocks: Select from available test provider results
-      - For text blocks: 
+      - **For test-driven blocks** — Select from available test provider results[^5]
+      - **For text blocks** — 
          
-         - For new block:
-            - Add new editable text content block
-            - Review and collaborate on the content block
-         - For existing blocks: Select from available texts from content provider
+         a. For new block:
+            1. Add new editable text content block
+            2. Review and collaborate on the content block
+         b. For existing blocks: Select from available texts from content provider
+
+2. [Submit your model documentation for review](/guide/model-documentation/submit-for-approval.qmd).
```

Comment:
This is under section 4. in the parent topic, so it should not be under the third section here.

Reply:
I'm a little confused, there isn't a section 4 in the original?